test/functional/crud_api_tests.js

```diff
@@ -1023,6 +1023,44 @@ describe('CRUD API', function() {
     }
   });
 
+  it('should correctly throw error if update doc for findOneAndUpdate lacks atomic operator', {
+    // Add a tag that our runner can trigger on
+    // in this case we are setting that node needs to be higher than 0.10.X to run
+    metadata: {
+      requires: { topology: ['single', 'replicaset', 'sharded', 'ssl', 'heap', 'wiredtiger'] }
```

Comment:
I believe we don't need to specify a topology, since we are running against all of them.